src/api/firebase.js

```diff
@@ -69,6 +69,8 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 		isChecked: false,
 		name: itemName,
 		totalPurchases: 0,
+		previousEstimate: null,
+		currentEstimate: parseInt(daysUntilNextPurchase),
```

Comment:
This seems like a good idea to make calculations easier but the problem with this is that it remains the same even though the days pass. For example:
- I buy an item today, November 17th and I'm estimated to buy it again in 14 days (December 1st). So the `dateNextPurchased` is going to be December 1 and the `currentEstimate` is going to be 14.
But if I visit my list in 10 days, I'll be supposed to buy the item in 4 days according to `dateNextPurchased` and 14 days according to `currentEstimate`. The accurate value is the one calculated between the current date and the `dateNextPurchased`, not the `currentEstimate`. 
You can use the `getDaysBetweenDates` function from the dates.js file to get the days between the current date and the `dateNextPurchased`

Reply:
Resolving this issue because my teammates already worked on this. :)